unit_scaling/functional.py

```diff
@@ -131,12 +132,9 @@ def matmul(
     left_grad_scale = right_size**-0.5
     right_grad_scale = left_size**-0.5
 
-    if constraint:
-        scale = constraint(output_scale, left_grad_scale, right_grad_scale)
-        if isinstance(scale, Sequence):
-            output_scale, left_grad_scale, right_grad_scale = scale  # type: ignore
```

Comment:
When did we previously return a sequence? Can't see any in `constraints`.

Reply:
Yeah I think I had a custom constraint that did this (back when that was allowed). Might leave in for now just to give the option of adding constraints later that do?